public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
```

Comment:
@arreche this also shows that you did not run the tests! Otherwise, they would've failed ;)